src_rebuild/GAME/C/SYSTEM.H

```diff
@@ -37,7 +37,7 @@ extern void sys_freeall();
 		char* _oldmalloc = mallocptr;
 
 #define MALLOC_END() \
-	if(mallocptr > _oldmalloc) printWarning("malloc(%d) in " __FUNCTION__ ", line %d. Malloc usage: %d\n", mallocptr-_oldmalloc, __LINE__, (mallocptr-mallocptr_start));\
+	if(mallocptr > _oldmalloc) printWarning("malloc(%d) in %s, line %d. Malloc usage: %d\n", __FUNCTION__, mallocptr-_oldmalloc, __LINE__, (mallocptr-mallocptr_start));\
```

Comment:
In GNU C, `__FUNCTION__` is not a macro, so we can't use it for string concatenation.